src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
+    private final double width;
+    private final double length;
+    private String color;
+
+    public Rectangle(double width, double length, String color) {
+        this.width = width;
+        this.length = length;
+        setColor(color);
+    }
```

Comment:
Creating a figure, don't pass expressions in the constructor. Instead, create separate variables for 'width', 'length', and 'color' and then pass them to the constructor for better code readability.